Addresses #5984 Doc: Making the negation operator clearer (#5985)
aiidateam/aiida-core: docs/source/topics/database.rst

| ``contains`` |    lists    | ``'attributes.some_key': {'contains': ['a', 'b']}``   | Filter for lists that should contain certain values.                         |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+

> As previously mentioned in the :ref:`section about filters<how-to:query:filters>` all the operator can be negated (``NOT`` operator) by adding either a ``!`` or ``~`` in front of the operator.

Reviewer: ```suggestion
As mentioned in the :ref:`section about filters<how-to:query:filters>` all operators can be turned into their associated negation (``NOT`` operator) by adding a ``!`` in front of the operator.
```

| ``contains`` |    lists    | ``'attributes.some_key': {'contains': ['a', 'b']}``   | Filter for lists that should contain certain values.                         |
+--------------+-------------+-------------------------------------------------------+------------------------------------------------------------------------------+

> As mentioned in the :ref:`section about operation negations<how-to:query:filters-operator-negations>` all operators can be turned into their associated negation (``NOT`` operator) by adding a ``!`` in front of the operator.

Reviewer: ```suggestion
As mentioned in the :ref:`section about operatior negations<how-to:query:filters:operator-negations>` all operators can be turned into their associated negation (``NOT`` operator) by adding a ``!`` in front of the operator.
```
